### ISSUE TITLE
createdAt pagination

### ISSUE THREAD

**Issue by michaelsharpe** [tmeasday/create-graphql-server#53]
Thanks so much for all the work you have put into this. It is making many things so much faster for my workflow.

I am hitting a snag though with the way pagination works.  It uses the createdAt field, which seems to be an ISO date format.  This means that the default, 0, is not pulling in any documents.  If I switch it to this:

```javascript
all({ lastCreatedAt = 0, limit = 10 }) {
    return this.collection.find({
      createdAt: { $gt: new Date(lastCreatedAt) },
    }).sort({ createdAt: 1 }).limit(limit).toArray();
  }
```

It works fine.  

I feel like there is something on my end I am not doing to make this work properly.  Any help or clarification would be greatly appreciated.

Thanks!

**Comment by tmeasday** [comment]
Hi @michaelsharpe -- I'm not sure; perhaps you entered the data outside of our mutations? Otherwise we definitely use `Float`s for dates on the GraphQL API and pretty much directly read and write the fields to the API directly.  Our `insert` calls just use `Date.now()` which inserts a float too: 

https://github.com/tmeasday/create-graphql-server/blob/ac948cdff143eba0fb50ebcde7f80108543c42fb/generate/model/templates/base.js.template#L24

**Comment by michaelsharpe** [comment]
Thanks for the quick response!  After some experimentation with the given mutations, I discovered that it was using a float.  I was working with a previous seed program I made that worked with Mongoose, and was setting both fields as Date types.  I am going to have to rewrite my seeds to work with the formatting for CGS.

Thanks again for making such a great tool, btw.  Your architecture is very well designed.
